xc7/fasm2bels/ioi_models.py

```diff
@@ -0,0 +1,96 @@
+from .verilog_modeling import Site  # , Bel
+
+
+def get_ioi_site(db, grid, tile, site):
+    """
+    Returns a prxjray.tile.Site object for given ILOGIC/OLOGIC/IDELAY site.
+    """
+
+    gridinfo = grid.gridinfo_at_tilename(tile)
+    tile_type = db.get_tile_type(gridinfo.tile_type)
+
+    site_type, site_y = site.split("_")
+
+    sites = tile_type.get_instance_sites(gridinfo)
+    sites = [s for s in sites if site_type in s.name]
+    sites.sort(key=lambda s: s.y)
+
+    if len(sites) == 1:
+        iob_site = sites[0]
+    else:
+        iob_site = sites[1 - int(site[-1])]
+
+    return iob_site
+
+
+def process_idelay(top, features):
+    # TODO: Support IDELAY
+    pass
```

Comment:
Problaly worth having an assert if [IDELAY.IN_USE](https://github.com/SymbiFlow/prjxray-db/blob/master/artix7/segbits_lioi3.db#L13) is set?

Reply:
Done.